src/components/AeAmount.vue

```diff
@@ -2,7 +2,10 @@
   <span class="ae-amount">
     <template>{{ roundedAmount }}</template>
     <!--eslint-disable-next-line vue-i18n/no-raw-text-->
-    <span class="ae">{{ tokenSymbol || 'AE' }}</span>
+    <span
+      v-if="!noSymbol"
+      class="ae"
+    >{{ tokenSymbol || 'AE' }}</span>
```

Comment:
```suggestion
    >
      {{ tokenSymbol || 'AE' }}
    </span>
```
I propose to avoid such structure

Reply:
It is difficult to have such agreement in a team until we have a corresponding linter rule. But I'm not a fan of adding custom rules, the `vue/recommended` should be fine.